modules/monitoring-configuring-persistent-storage.adoc

```diff
@@ -15,9 +15,9 @@ Running cluster monitoring with persistent storage means that your metrics are s
 
 * Verify that you have a persistent volume (PV) ready to be claimed by the persistent volume claim (PVC), one PV for each replica. Because Prometheus and Alertmanager both have two replicas, you need four PVs to support the entire monitoring stack. The PVs are available from the Local Storage Operator, but not if you have enabled dynamically provisioned storage.
 
-* Use the block type of storage.
-
+* Use `Filesystem` as the storage type for `volumeMode` when configuring the persistent volume.
```

Comment:
@rolfedh Thanks for the suggestion! However, I think this level of detail isn't necessary in this context. However, to make the sentence centered more around user action, I can changing the sentence to read "Use `Filesystem` as the storage type for `volumeMode` when you configure the persistent volume." WDYT?

Reply:
I like the simplicity. What about our guidelines on labeling the objects between backticks? Will the user know where to find `volumeMode`?

Reply:
Thanks, @rolfedh! Maybe this would be better: "Use `Filesystem` as the storage type for the `volumeMode` parameter when you configure the persistent volume." WDYT?